sql/pipelette.sql

```diff
@@ -13,13 +13,26 @@ CREATE TABLE `user` (
 CREATE TABLE `playlist` (
     `id` INT NOT NULL AUTO_INCREMENT,
     `title` VARCHAR(255) NOT NULL,
-    PRIMARY KEY (`id`)
+    `UserId` INT NOT NULL,
```

Comment:
perfect :)

just for the case, usual convention for sql is to use snake_case: user_id instead of UserId